mobile/mobile.scss

```diff
@@ -1,8 +1,7 @@
 .d-header {
   display: grid;
-  grid:
-"left-menu logo extra-info chat search right-menu" auto;
-  grid-column-gap: 0.2em;
+  grid-template-areas: 'left-menu logo custom-header-icon-link-1 custom-header-icon-link-2 custom-header-icon-link-3 custom-header-icon-link-4 custom-header-icon-link-5 extra-info chat search right-menu';
+  grid-template-columns: auto auto auto auto auto auto auto auto auto auto;
```

Comment:
Can we use a repeat here? https://developer.mozilla.org/en-US/docs/Web/CSS/repeat